build.sbt

```diff
@@ -173,4 +165,4 @@ lazy val benchmark = project.in(file("benchmark"))
   .enablePlugins(JmhPlugin)
 
 lazy val all =
-  Seq(parser, util, ast, supportArgonaut, supportJson4s, supportPlay, supportRojoma, supportRojomaV3, supportSpray)
+  Seq(parser, util, ast, supportArgonaut, supportJson4s, supportPlay, supportSpray)
```

Comment:
We made automatic fully automated publishing work on http4s with a missing boopickle module, but it took a couple days and was totally not worth it.